src/components/AnnotationCreator.jsx

```diff
@@ -40,6 +43,13 @@ async function createEncounter(sightingData, addEncounterToSighting) {
     encounter => encounter?.guid,
     [],
   );
+  const encountersWithNoAnnotations = filter(
+    get(sightingData, 'encounters'),
```

Comment:
fall back to an empty array or you will get an error if sightingData.encounters does not exist